cytoscape-supportimages.js

```diff
@@ -734,7 +734,12 @@
                 } else {
                   context.drawImage(img, supportImage.bounds.x, supportImage.bounds.y, supportImage.bounds.width, supportImage.bounds.height);
                 }
-                requestAnimationFrame(animateFrames);
+
+                if(r.data.supportImageExt._private.supportImages.some(img => img.id === supportImage.id && img.visible === true)){
```

Comment:
This should be verified in the `removeSupportImage` and `setImageVisible`, since we already have the `img` reference there, so we don't need to loop through all images.

Please, set a control variable in those functions and check it here.